app/js/views/themes.jsx

```diff
@@ -51,24 +30,12 @@ export default class Themes extends React.Component {
   }
 
   render() {
-    const themeItems = this.state.themes.map((theme, id) => {
-      let itemClassName = 'themes-list__item';
-      if (!theme.enabled) {
-        itemClassName += ' themes-list__item--deactivated';
-      }
-
-      return (
-        <li key={id}
-            className={itemClassName}>
-          <input className="themes-list__toggle" type="checkbox"
-                 checked={theme.enabled}
-                 onChange={this.handleOnChange.bind(this, id)}/>
-          <span className="themes-list__name">{theme.name}</span>
-          <button className="themes-list__remove"
-                  onClick={this.handleOnDelete.bind(this, id)}></button>
-        </li>
-      );
-    });
+    const themeItems = this.state.themes.map(theme => (
+      <ThemesListItem key={theme.id}
+                      theme={theme}
+                      update={this.update}
```

Comment:
I'm not sure what's the recommended workflow for inter-component communication in React, but it feels natural for the ```ThemesListItem``` to expose event (```onDeleted```? I see it's only used for this right now) instead that we subscribe for in this component and do the action instead passing functions back and forth.

What do you think?